src/main/resources/db/changelog/changes01/13072021_add_column_dis_viewed.yml

```diff
@@ -0,0 +1,11 @@
+databaseChangeLog:
+  - changeSet:
+      id: 13072021_add_column_dis_viewed
+      author: tesliuk.ua@gmail.com
+      changes:
```

Comment:
we should add the precondition in order to check that the added column doesn't already exist

Reply:
fixed